### DISCUSSION TITLE
Windows 11 UTM on M1 pro "Let's connect you to a network"

### DISCUSSION THREAD

**Discussion by Ramez-Dous** [utmapp/UTM#3870] [General]
Hello,

I'm stuck with this error when installing Windows 11 on my MacBook Pro 16 "M1 Pro". 

"
Let's connect you to a network
You'll need an internet connection to
continue setting up your device. Once
connected, you'll get the latest features
and security updates.

Having trouble getting connected?
For troubleshooting tips, use another
device and visit aka.ms/networksetup 
"

Tried deleting and creating the VM many times. Also, tried Shared Network, Bridged, and Emulated VLAN. Tried many times with different Emulated Network Card. Also, Tried on WiFi and a USB to Ethernet. Nothing seems to work.

Any ideas?!

Thank you!

**Comment by kotx** [comment]
Download the SPICE Guest Tools ISO (https://mac.getutm.app/support/) and attach it to the VM. Then press Shift+F10 in the VM to open a command prompt. Go to `D:\` by typing `D:` and run the setup exe located in the ISO. If you reboot you'll see a network connection has been added.
Also IIRC Shared Network doesn't work, I had to switch to Bridged.

**Reply by brbell01** [reply]
How to boot back into my configured VM using Bridged?  It boots to the windows setup now?

**Comment by anikdoshi2003** [comment]
How to run the setup?

**Reply by Ramez-Dous** [reply]
https://github.com/utmapp/UTM/wiki/Install%20Windows%20ARM64%20on%20Apple%20M1

**Comment by ilu2k21** [comment]
Installed guest tools but still get the lets connect you to a network message, any hints`?
There was no setup.exe instead there was a spice-guest-toolsxxxx.exe

**Reply by Ramez-Dous** [reply]
Just run the .exe file.

**Comment by Ramez-Dous** [comment]
> Download the SPICE Guest Tools ISO (https://mac.getutm.app/support/) and attach it to the VM. Then press Shift+F10 in the VM to open a command prompt. Go to `D:\` by typing `D:` and run the setup exe located in the ISO. If you reboot you'll see a network connection has been added. Also IIRC Shared Network doesn't work, I had to switch to Bridged.

Thank you!
This solved the issue for me.
Shared Network works for me. Emulated Network Card = virtio-net-pci

**Comment by kotx** [comment]
> Installed guest tools but still get the lets connect you to a network message, any hints`?

Try rebooting the VM?

**Comment by conath** [comment]
According to UTM users on the Discord server, typing `oobe\bypassnro` into the command prompt (fn-shift-F10) reboots the VM and enables a “Skip” button on this step of the setup.

**Reply by ilu2k21** [reply]
This doesnt work either, i get this message: 'oobe' is not recognized as an internal or external command, operable program or batch file.

**Reply by conath** [reply]
Did you put backslash? `oobe\bypassnro`

**Reply by ilu2k21** [reply]
nope i didnt, that did the job.

Thank you

**Comment by rebelopz** [comment]
OMG, this has saved my life, thank you. I was able to run setup from the commandline and finally got online prior to installing. Thanks a lot, guys!

**Comment by neurolabusc** [comment]
Can I suggest this hint is included in the [UTM documentation for installing Windows](https://docs.getutm.app/guest-support/windows/).

**Comment by rev111** [comment]
I'm on an M1 Mac Air and stuck at the network dialog of Win 11, build 22598. Shift+F10 will show the volume mute symbol on the Mac, Shift+Fn+F10 will show the Windows "wait circle" for a microsecond, then nothing happens. No command prompt is showing. I tried many combinations of control, option, F9 and F11. I tried it with an external keyboard. The keys are working in the VM, I changed display resolution in the BIOS where I had to save with the (Fn+)F10 key. But in the network dialog nothing happens. Does anyone have an idea what could be wrong?

EDIT: The solution was to **press Alt+Tab (Option+Tab on the Mac)** to switch to a window in the background. The command prompt is on that window.

**Reply by luchi101** [reply]
on the window setup "lets connect you to a network" 

“Shift + F10” keyboard shortcut. In Command Prompt, type the OOBE\BYPASSNRO command to bypass network requirements on Windows 11 and press Enter.

**Comment by luchi101** [comment]
> Hello,
> 
> I'm stuck with this error when installing Windows 11 on my MacBook Pro 16 "M1 Pro".
> 
> " Let's connect you to a network You'll need an internet connection to continue setting up your device. Once connected, you'll get the latest features and security updates.
> 
> Having trouble getting connected? For troubleshooting tips, use another device and visit aka.ms/networksetup "
> 
> Tried deleting and creating the VM many times. Also, tried Shared Network, Bridged, and Emulated VLAN. Tried many times with different Emulated Network Card. Also, Tried on WiFi and a USB to Ethernet. Nothing seems to work.
> 
> Any ideas?!
> 
> Thank you!

“Shift + F10” keyboard shortcut. In Command Prompt, type the OOBE\BYPASSNRO command to bypass network requirements on Windows 11 and press Enter.
